specification/resources/resource-manager/Microsoft.Authorization/preview/2023-03-01-preview/privateLinks.json

```diff
@@ -0,0 +1,832 @@
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List provider operations": {
+            "$ref": "./examples/ListProviderOperations.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PrivateLinkAssociationObject": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkAssociationProperties",
+          "description": "The properties of the PrivateLinkAssociation."
+        }
+      }
+    },
+    "EnablePrivateLinkNetworkAccessObject": {
+      "type": "object",
+      "properties": {
+        "scope": {
+          "type": "string",
+          "description": "The scope of private link associations where network access is updated."
+        }
+      }
+    },
+    "PrivateLinkAssociationProperties": {
+      "type": "object",
+      "properties": {
+        "privateLink": {
+          "type": "string",
+          "description": "The rmpl Resource ID."
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "publicNetworkAccessOptions",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PrivateLinkAssociationGetResult": {
+      "description": "Result of the request to get PLA for a MG scope. ",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkAssociation"
+          },
+          "description": "private link association information."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of resource information. Call ListNext() with this to fetch the next page of resource information."
+        }
+      }
+    },
+    "PrivateLinkAssociationPropertiesExpanded": {
+      "type": "object",
+      "properties": {
+        "privateLink": {
+          "type": "string",
+          "description": "The rmpl Resource ID."
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "publicNetworkAccessOptions",
+            "modelAsString": true
+          }
+        },
+        "tenantID": {
+          "type": "string",
+          "description": "The TenantID."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope of the private link association."
+        }
+      },
+      "description": "Private Link Association Properties."
+    },
+    "PrivateLinkAssociation": {
```

Comment:
It was present on version 2020-05-01. Not sure if I remove it now it will be marked as breaking change. Not changing thses APIs in the 2023 version. Only new API added in this version is POST "/providers/Microsoft.Authorization/enablePrivateLinkNetworkAccess" everything else is same,